tests/console/yast2_apparmor.pm

```diff
@@ -18,7 +18,7 @@ use utils qw(zypper_call systemctl);
 use version_utils qw(is_pre_15 is_sle is_opensuse is_leap);
 
 sub install_extra_packages_requested {
-    check_screen([qw(yast2_apparmor_extra_packages_requested yast2_apparmor)], 30);
+    check_screen([qw(yast2_apparmor_extra_packages_requested)], 30);
```

Comment:
If you want to remove multi tag assert, then remove array wrapper and put it directly in the line below instead of `match_has_tag`. Also, be aware, that now, for the cases where there is no screen for extra packages we will waste 30 seconds on waiting for it.

Reply:
thanks! far points!